packages/core/client.js

```diff
@@ -205,13 +207,11 @@ class BugsnagClient {
       }
 
       // only leave a crumb for the error if actually got sent
-      if (this.config.autoBreadcrumbs) {
-        this.leaveBreadcrumb(event.errorClass, {
-          errorClass: event.errorClass,
-          errorMessage: event.errorMessage,
-          severity: event.severity
-        }, 'error')
-      }
+      BugsnagClient.prototype.leaveBreadcrumb.call(this, event.errorClass, {
```

Comment:
Node's notifier monkey-patches the `leaveBreadcrumb()` method to log a warning that breadcrumbs aren't supported.

Calling the method via the prototype ensures "internal" calls to `leaveBreadcrumb()` don't trigger the log. I can explain more if this doesn't make sense!